impala/dbapi.py

```diff
@@ -101,6 +101,8 @@ def connect(host='localhost', port=21050, database=None, timeout=None,
         Currently cookie retention is supported for GSSAPI/LDAP/SASL/NOSASL/JWT over http.
     jwt: string containing a JSON Web Token
         This is used for auth_mechanism=JWT when using the HTTP transport.
+    user_agent: A user specified user agent when HTTP transport is used. If none is specified,
+        'Python/ImpalaHttpClient' is used
```

Comment:
At the moment  'Python/ImpalaHttpClient' is used as the user agent for both impala shell and impyla. This was caused by code copying (maybe by me), but I think it's wrong, they should be different.  I think documenting that the user agent is 'Python/ImpalaHttpClient' would then make changing the default to be a breaking change, if we are being pedantic. So, please can you EITHER not document the default value here, OR change the default value to be something different (perhaps 'Python/ImpylaHttpClient').

Reply:
Thanks for reviewing @bartash 
I have updated the default user agent to 'Python/ImpylaHttpClient' as per suggestion, which I felt was more correct thing to do.